### ISSUE TITLE
Error exporting to HTML

### ISSUE THREAD

**Issue by kevindoveton** [careteditor/caret#497]
The following error has occurred:
```
{
    "message": "Cannot read property 'remove' of null",
    "version": "3.4.2",
    "os": "macOS Sierra",
    "path": "extensions/Preview/renderer.js",
    "code": "(r.nextElementSibling.remove(),r.previousElementSibling.remove())}function normalizeLinks(e)",
    "line": 5,
    "column": 2072,
    "stack": [
        "at safe (extensions/Preview/renderer.js:5:2072)",
        "at Object.module.exports.getNormalizedMarkdownElement (extensions/Preview/renderer.js:5:3090)",
        "at Preview.render (extensions/Preview/Preview.js:96:36)",
        "at Preview._getExportHTML (extensions/Preview/Preview.js:209:12)",
        "at Preview.exportToHTML (extensions/Preview/Preview.js:168:37)",
        "at t.value (web/Extension.Command.js:5:2411)",
        "at _0x34d3x3.value (Caret.js:5:23563)",
        "at emitThree ((events.js:116:13)",
        "at EventEmitter.emit ((events.js:197:7)"
    ]
}
```

**Comment by kevindoveton** [comment]
Exporting to PDF does not work either - but didn't display an error message

Can provide the document privately if needed.

**Comment by kevindoveton** [comment]
did a bit more debugging - error occurs when using the following 

```
[[toc]]
---
```

separating the lines fixes the issue 

```
[[toc]]

---
```

**Comment by astoilkov** [comment]
Awesome. Thanks for the additional information. It helped a lot. We will investigate the problem and fix it.

**Comment by astoilkov** [comment]
This issue is now fixed in the new Caret 4.0 Beta available [here](https://caret.io/4beta/).

Closing this issue. Thanks for reporting.
